items.json

```diff
@@ -5995,6 +5995,42 @@
       }
     ]
   },
+  {
+    "id": "crpg_bolt_f",
+    "name": "Heavy Piercing Bolts",
+    "culture": "Empire",
+    "type": "Bolts",
+    "price": 7815,
+    "weight": 0.12,

```

Comment:
for some reason my previous comment was cut. I said 0.3 would be the minimum i'd accept